fem/fe/fe_fixed_order.hpp

```diff
@@ -1018,6 +1018,34 @@ class Nedelec1PyrFiniteElement : public VectorFiniteElement
 };
 
 
+/// A 3D 2nd order Nedelec element on a pyramid
+class Nedelec2PyrFiniteElement : public VectorFiniteElement
```

Comment:
Should we keep this in the PR? It's a very element and the test has been commented out: https://github.com/mfem/mfem/pull/4460/files#diff-810b21fbfced27e1db37060dd1f22d060a494cfb7e6695aba41f95fd518bd357R178

Reply:
A fair question. This was added by request to aid in testing before the arbitrary order version was available. We probably no longer need it. If no one expresses a desire to keep it, I'll remove it before merging.